### PR TITLE
ksphere to dkp

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/dispatch/1.1/release-notes/open-source-attribution/index.md [hunk 1]
```diff
@@ -146,6 +146,6 @@ Software that are utilized by D2IQ Dispatch.
...
-| golang-lru     | https://packages.d2iq.com/ksphere/golang-lru-0.5.4.tar.gz    | [Mozilla Public License 2.0](https://github.com/hashicorp/golang-lru/blob/v0.5.4/LICENSE)    |
-| errwrap        | https://packages.d2iq.com/ksphere/errwrap-1.0.0.tar.gz       | [Mozilla Public License 2.0](https://github.com/hashicorp/errwrap/blob/v1.0.0/LICENSE)       |
-| go-multierror  | https://packages.d2iq.com/ksphere/go-multierror-1.1.0.tar.gz | [Mozilla Public License 2.0](https://github.com/hashicorp/go-multierror/blob/v1.1.0/LICENSE) |
+| golang-lru     | https://packages.d2iq.com/dkp/golang-lru-0.5.4.tar.gz    | [Mozilla Public License 2.0](https://github.com/hashicorp/golang-lru/blob/v0.5.4/LICENSE)    |
+| errwrap        | https://packages.d2iq.com/dkp/errwrap-1.0.0.tar.gz       | [Mozilla Public License 2.0](https://github.com/hashicorp/errwrap/blob/v1.0.0/LICENSE)       |
+| go-multierror  | https://packages.d2iq.com/dkp/go-multierror-1.1.0.tar.gz | [Mozilla Public License 2.0](https://github.com/hashicorp/go-multierror/blob/v1.1.0/LICENSE) |
```
Comment (dkoshkin) [inline pages/dkp/dispatch/1.1/release-notes/open-source-attribution/index.md:151]: Im not sure what these packages are, but they are not available in `https://packages.d2iq.com/dkp/golang-lru-0.5.4.tar.g`

Comment (cmastr) [reply pages/dkp/dispatch/1.1/release-notes/open-source-attribution/index.md]: yes, got overzealous. Changed back.

**Review by dkoshkin — APPROVED** [review]

**Comment by samvantran** [comment]
Let's make sure this PR can deploy a working site, will need this build to pass `continuous-integration/jenkins/pr-merge`

**Comment by pierrebeitz** [comment]
i started all over, as i could not track what exactly has been done before and wanted to make sure we don't change things accidentally (like those packages-links above). you can have a look at what changed by reviewing commit-by-commit. there might be more clever ways to get to the final result, but i tried for a very reproducible series of steps (note the `--accept-all` in almost any commit) so i went for rather "small" but safe-looking changesets.

for reviewing: the commit saying "add redirect" and the last one should get extra attention, please!

**Comment by cmastr** [comment]
> Let's make sure this PR can deploy a working site, will need this build to pass `continuous-integration/jenkins/pr-merge`

working!

**Review by samvantran — APPROVED** [review]
`1607` files changed 😨 

but since build site looks good, i'm 👍

**Review by dkoshkin — APPROVED** [review]

**Pull request merged**